server/webapp/WEB-INF/rails/webpack/views/pages/config_repos/config_repos_widget.tsx

```diff
@@ -170,12 +189,18 @@ class ConfigRepoWidget extends MithrilViewComponent<ShowObjectAttrs<ConfigRepo>>
       );
     }
 
+    const configRepoHasErrors = maybeWarning || (vnode.attrs.obj.lastParse() && vnode.attrs.obj.lastParse()!.error());
+
     return (
-      <CollapsiblePanel header={<HeaderWidget repo={vnode.attrs.obj} pluginInfos={vnode.attrs.pluginInfos}/>}
+      <CollapsiblePanel styleClass={configRepoHasErrors ? styles.configRepoErrorHeader : ""}
+                        header={<HeaderWidget repo={vnode.attrs.obj} pluginInfos={vnode.attrs.pluginInfos}/>}
+                        dataTestId={"config-repo-details-panel"}
                         actions={actionButtons} expanded={vnode.attrs.index === 0}>
-        {maybeWarning}
-        {lastParseRevision}
-        <KeyValuePair data={allAttributes}/>
+        {maybeWarning ? <div class={styles.errorMessage}>{maybeWarning}</div> : null}
```

Comment:
The `div` could be added to where `maybeWarning` is defined. It would improve readability IMO.